src/plot/xy-plot.js

```diff
@@ -545,11 +545,13 @@ class XYPlot extends React.Component {
       );
     }
     const components = this._getClonedChildComponents();
+    const xyPlotStyle = style ? style.XYPlot : {};
     return (
       <div
         style={{
           width: `${width}px`,
-          height: `${height}px`
+          height: `${height}px`,
+          ...xyPlotStyle
```

Comment:
This was needed in order to pass in the negative `z-index`. The "empty" xy-plot puts the `...style` directly on the div. That could be done here, but I thought that could be dangerous, so I created a new style specifically for this div.